hw/top_earlgrey/dv/chip_rom_tests.hjson

```diff
@@ -881,6 +881,23 @@
       run_timeout_mins: 480
     }
 
+    // ROM self hash test.
+    {
+      name: rom_e2e_self_hash
+      uvm_test_seq: chip_sw_base_vseq
+      sw_images: [
+        "//sw/device/silicon_creator/rom/e2e:rom_e2e_self_hash:1:signed"
+        "//hw/ip/otp_ctrl/data:img_test_unlocked0:4",
+      ]
+      // TODO(#18686): update to use ROM with real keys
+      en_run_modes: ["sw_test_mode_rom_with_fake_keys"]
```

Comment:
```suggestion
      en_run_modes: ["sw_test_mode_rom_with_real_keys"]
```

Reply:
Yes please see my TODO, I was planning on updating this once we gen real keys.